cmd/gpud/command/command.go

```diff
@@ -52,6 +48,8 @@ var (
 	ibstatCommand string
 
 	checkInfiniBand bool
+
+	deprecatedWebEnable bool
```

Comment:
No one references it, shall we remove it ?

Reply:
Some GPUd commands references it. After 0.5 release, we can drop this :)